requirements/requirements.txt

```diff
@@ -4,8 +4,7 @@ aiohttp==3.6.2
     # via -r /awx_devel/requirements/requirements.in
 aioredis==1.3.1
     # via channels-redis
-ansible-runner==2.0.1
-    # via -r /awx_devel/requirements/requirements.in
+    # via -r /awx_devel/requirements/requirements_git.txt
```

Comment:
This line seems wrong.

Reply:
This is an unfortunate side effect of the `pip-compile` part of the updater.sh script, and the fact that we remove all `git+` dependencies it adds to `requirements.txt`.  

The pattern so far has just been to leave this in (that's what we did for certifi it looks like).

Reply:
oh... so it gets installed by a separate `pip install -r ...` command?

Reply:
@AlanCoding yes, this happens here upstream - https://github.com/ansible/awx/blob/devel/Makefile#L139